app/Console/Commands/UpdateProcessedJobsTopicNumber.php

```diff
@@ -41,8 +41,17 @@ public function handle()
     {
         $getAllProcessedJobs = ProcessedJob::select('id', 'response')->orderBy('id', 'desc')->get();
```

Comment:
variable name should not start with a verb, please change it to processedJobs or allProcessedJob